### ISSUE TITLE
Add support for IonSchema?

### ISSUE THREAD

**Issue by cameronsstone** [amzn/ion-intellij-plugin#7]
**[Feature Request]**: add support for IonSchema files?

You've got the spec and a kotlin parser. It'd be great to have the additional highlighting, and syntax error detection.

Stretch goal: support navigation between types. 🥺

**Comment by jayessdeesea** [comment]
This would help me validate my data is well formed without having to write code

**Comment by hsbakshi** [comment]
Yes, please also include support for files with the *.isl file extension. This extension is used in the examples in the [Ion spec](https://amazon-ion.github.io/ion-schema/docs/isl-2-0/spec).

**Comment by r0b0ji** [comment]
You can already do this by associating .isl file type with Ion plugin.

1. Configure: Settings | Editor | File Types
2. In the list of Recognized File Types find, Ion language file and click that
3. In the file name patterns, add `*.isl`

**Comment by dlurton** [comment]
The association suggested by @r0b0ji should be added by default, IMO.
